lib/client-assets.php

```diff
@@ -653,8 +653,15 @@ function gutenberg_default_script_modules() {
 				break;
 		}
 
+		// All script modules in Gutenberg are (currently) related to the Interactivity API which prioritizes server-side rendering.
+		// Therefore, the modules should be fetched with a low priority to avoid network contention with any LCP element resource.
+		// For allowing a block to opt-in to another fetchpriority, see <https://github.com/WordPress/gutenberg/issues/71366>.
```

Comment:
```suggestion
		/*
		 * All script modules in Gutenberg are (currently) related to the Interactivity API which prioritizes server-side rendering.
		 * Therefore, the modules should be fetched with a low priority to avoid network contention with any LCP element resource.
		 * For allowing a block to opt-in to another fetchpriority, see <https://github.com/WordPress/gutenberg/issues/71366>.
		 */
```

Nit. Use multiline comment.